packages/plugin-dev/src/browser/hosted-plugin-manager-client.ts

```diff
@@ -128,13 +128,6 @@ export class HostedPluginManagerClient {
         }
     }
 
-    get lastPluginLocation(): string | undefined {
```

Comment:
Why remove the method?

Reply:
it is not used. should i keep it anyway?

Reply:
It might not be used by us, but someone else (extender) may be using it.
If you do remove it, I'd make a note in the changelog.

Reply:
Ok. I reverted the deletion. Maybe some extender uses this API.